docs/plugins/haveibeenpwned/plugin.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * KeeWeb plugin: haveibeenpwned
+ * @author Olivier LEVILLAIN
+ * @license MIT
+ */
+
+const DetailsView = require("views/details/details-view");
+const Alerts = require("comp/alerts");
+const Logger = require('util/logger');
+const InputFx = require('util/input-fx');
+
+const detailsViewFieldChanged = DetailsView.prototype.fieldChanged;
+
+DetailsView.prototype.checkPwnedPwd = false;
+DetailsView.prototype.checkPwnedName = false;
+DetailsView.prototype.blockPwnedPwd = false;
+DetailsView.prototype.blockPwnedName = false;
+DetailsView.prototype.logger = new Logger("HaveIBeenPwned");
+
+DetailsView.prototype._alert = function (msg) {
```

Comment:
This method can me easily implemented as a simple function outside `DetailsView` as well.

Reply:
moved to utils class